traceabilityViewer/traceabilityViewer/settings.py

```diff
@@ -0,0 +1,136 @@
+"""
+Django settings for traceabilityViewer project.
+
+Generated by 'django-admin startproject' using Django 4.2.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+from neomodel import config
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-9@1+y@2v04x9t4r=ih5-%j7ueh(84j6($3tt6js&pd%ozl91du"
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = ["localhost", "127.0.0.1", "0.0.0.0"]
+
+# config.DATABASE_URL = "bolt://neo4j:password@localhost:7687"
+config.DATABASE_URL = "bolt://neo4j:password@neo4j_db:7687"
```

Comment:
This should become configurable.

Reply:
Resolved in 0b18671faf1744006b762826255bb4df5a76b926